azkaban-common/src/main/java/azkaban/project/NodeBeanLoader.java

```diff
@@ -71,6 +72,7 @@ public AzkabanNode toAzkabanNode(final NodeBean nodeBean) {
           .setDependsOn(nodeBean.getDependsOn())
           .setNodes(
               nodeBean.getNodes().stream().map(this::toAzkabanNode).collect(Collectors.toList()))
+          .setFlowTrigger(toFlowTrigger(nodeBean.getTrigger()))
```

Comment:
Nit: In fact, if we use build pattern, we don't need to explicitly call this method with "set{blabla}", because almost every method other than `build` should be called `set`.

Reply:
Thanks for the suggestion. I can remove "set" in the method name.